src/pymovements/gaze/gaze_dataframe.py

```diff
@@ -883,6 +889,55 @@ def detect(
                 how='diagonal',
             )
 
+    def compute_event_properties(
+            self,
+            event_properties: str | tuple[str, dict[str, Any]]
+            | list[str | tuple[str, dict[str, Any]]],
+            name: str | None = None,
+            verbose: bool = True,
+    ) -> pm.EventDataFrame:
```

Comment:
there are inconsistencies between the return types of `Dataset` and `GazeDataFrame` methods.
`Dataset` inplace methods return `Dataset`, the one from `GazeDataFrame` return `None`.

We will resolve this inconsistency sometime in the future when we decide which inplace return type we want to favor.

For the time being, it would be best to stay as consistent as possible with the particular modules to not introduce any further confusion, i.e. I would prefer returning `None` here

Reply:
haha, okay, I didn't notice that; yes let's keep inconsistencies consistent